### ISSUE TITLE
grunt start failed on mac OS X

### ISSUE THREAD

**Issue by Tuxik** [butterproject/butter#32]
while installing on mac OS X following instruction of the README.md I have this error : 
```
>> /bin/sh: build/cache/mac/0.12.3/nwjs.app/Contents/MacOS/nwjs: No such file or directory
>> Exited with code: 127.
Warning: Task "exec:mac" failed. Use --force to continue.

Aborted due to warnings.
```

**Comment by edospadoni** [comment]
file Gruntfile.js line 234.
change this: 
`cmd: 'build/cache/mac/<%= nwjs.options.version %>/nwjs.app/Contents/MacOS/nwjs .'`
to this:
`cmd: 'cache/<%= nwjs.options.version %>/osx64/nwjs.app/Contents/MacOS/nwjs .'`

**Comment by bvandorp** [comment]
edospadoni's solution worked for me

**Comment by MrAdder** [comment]
Fixed in PR https://github.com/butterproject/butter/pull/34

**Comment by xaiki** [comment]
fixed in #34
